Description: remove fullscreen
 the app start only in fullscreen
 that could be annoying on setups with multiple screens, so we start in windowed mode
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-04-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/setup.xml
+++ b/setup.xml
@@ -2,7 +2,7 @@
 	<archive path="./data/gui/"/>
 	<archive path="zip://vintageII.dat"/>
 	<time start="0" loop="off"/>
-	<display width="1280" height="720" fullscreen="on" title="Vintage II"/>
+	<display width="1280" height="720" fullscreen="off" title="Vintage II"/>
 	<music filename="vintageII.ogg" mute="off"/>
 	<osd show="on" location="bottom" delay="2.0"/>
 	<script filename="script.xml"/>
